views/showcase.html

```diff
@@ -18,7 +18,7 @@
         <a class="breadcrumb_link" href="/search">{{__('Data')}}</a>
       </li>
       <li class="breadcrumb_item">
-        <a class="breadcrumb_link" href="/search/{{ dataset.name }}" aria-current="page">{{ dataset.title or dataset.name }}</a>
+        <a class="breadcrumb_link" href="/{{ dataset.organization.name }}/{{ dataset.name }}" aria-current="page">{{ dataset.title or dataset.name }}</a>
```

Comment:
I'd recommend using `owner.name` instead of `dataset.organization.name`

Reply:
I followed your recommendation and added changes from `dataset.organization.name` to `owner.name`